src/silkaj.py

```diff
@@ -27,6 +27,7 @@ def usage():
     \n     [--comment=<comment>] \
     \n     [--allSources] \
     \n     [--outputBackChange=<public key[:checksum]>] \
+    \n     --yes, don't ask for prompt confirmation \
```

Comment:
I think a simple "if (c.contains_switches('yes') or c.contains_switches('y')) or ..." is the trick.
See there: https://github.com/chrissimpkins/commandlines (in the table "Arguments" there is exactly the same example).